app/views/games/index.html.erb

```diff
@@ -1,182 +1,51 @@
 <div class="container-fluid">
-  <section id="register">
-    <div class= "row">
+  <section id="games-list">
+    <div class="row">
       <div class="col-sm-5 col-sm-offset-1">
```

Comment:
I figured that was the case. The issue with this strategy is that it's only emulating a center and not actually providing it.

Does a `col-sm-6 text-center` not achieve the desired results?